channel.go

```diff
@@ -269,16 +274,34 @@ func TruncateWithMessage(message *Message) func(*truncateOptions) {
 	}
 }
 
+func TruncateWithUserID(userID string) func(*truncateOptions) {
+	return func(o *truncateOptions) {
+		o.UserID = userID
+	}
+}
+
+func TruncateWithUser(user *User) func(*truncateOptions) {
+	return func(o *truncateOptions) {
+		o.User = user
+	}
+}
+
 func TruncateWithTruncatedAt(truncatedAt *time.Time) func(*truncateOptions) {
 	return func(o *truncateOptions) {
 		o.TruncatedAt = truncatedAt
 	}
 }
 
+type TruncateResponse struct {
+	Response
+	Channel *Channel `json:"channel"`
+	Message *Message `json:"message"`
+}
+
 // Truncate removes all messages from the channel.
 // You can pass in options such as hard_delete, skip_push
 // or a custom message.
-func (ch *Channel) Truncate(ctx context.Context, options ...TruncateOption) (*Response, error) {
+func (ch *Channel) Truncate(ctx context.Context, options ...TruncateOption) (*TruncateResponse, error) {
```

Comment:
this is better response but breaking 🤦🏻‍♂️

Reply:
✅